app/lib/tag_manager.rb

```diff
@@ -32,6 +32,7 @@ def same_acct?(canonical, needle)
 
   def local_url?(url)
     uri    = Addressable::URI.parse(url).normalize
+    return false unless uri.host
```

Comment:
Hm, can that actually occur?

Reply:
it happened on a `dweb:/` link

Reply:
ok

Reply:
This breaks the tests, looking into it.

Reply:
Yeah, I guess we *could* fix the tests, but that added line doesn't actually prevent any crash, host being nil is tested down the line and isn't an issue.

Reply:
Sorry, I misunderstood what was happening and read that part of the code too fast. The test failures was because of me not resetting the `web_domain` properly in the tests. Fixed.